src/test.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './classes/person'
```

Comment:
- [ ] There is an error you need to this in your app as you can see in the screenshot below ⬇️  Please be sure to fix it in order to make your app works properly 👍 

![error](https://user-images.githubusercontent.com/6615394/223539616-3049c307-0f89-4b60-9b59-4358acd326d5.PNG)